src/server.rs

```diff
@@ -1,22 +1,20 @@
-use crate::auth;
-use crate::config::Config;
-use crate::file::Directory;
-use crate::header::{self, ContentDisposition};
-use crate::mime as mime_util;
-use crate::paste::{Paste, PasteType};
-use crate::util;
-use crate::AUTH_TOKEN_ENV;
+use crate::{
+    auth,
+    config::Config,
+    file::Directory,
+    header::{self, ContentDisposition},
+    mime as mime_util,
+    paste::{Paste, PasteType},
+    util, AUTH_TOKEN_ENV,
+};
```

Comment:
Can we revert this formatting change please? It is not relevant.

Reply:
Ah my bad, my editor is configured to squish the formatting like so. I'll revert shortly.

Reply:
Fixed [29857da](https://github.com/orhun/rustypaste/pull/46/commits/29857dac5cdb6b295134204f984a6485aa1e69dd)